src/Command/CleanAssetCommand.php

```diff
@@ -66,13 +66,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             foreach ($hashes as $hash) {
                 $usedCounter = $revRepo->hashReferenced($hash['hash']);
                 if ($usedCounter === 0) {
-                    /**@var StorageInterface $storage */
-                    foreach ($this->fileService->getStorages() as $storage) {
-                        $storage->remove($hash['hash']);
-                    }
```

Comment:
The remove is not necessary anymore?

Reply:
:-) Glad you ask. 

If a file is referenced by 2 ems sharing the same storage you will delete it for both. If you want to clean you storage you should create another storage and ems:asset: synch it (and drop the old storage). As only referenced file will be synched! All the rest will be cleaned. I will add a note in the doc about this.

Reply:
Okay perfect! 👍